tracer/src/Datadog.Trace/Tagging/AzureServiceBusTags.cs

```diff
@@ -26,6 +26,9 @@ internal partial class AzureServiceBusTags : OpenTelemetryTags
         [Tag(Trace.Tags.MessagingDestinationName)]
         public string MessagingDestinationName { get; set; }
 
+        [Tag(Trace.Tags.LegacyMessageBusDestination)]
+        public string LegacyMessageBusDestination { get; set; }
```

Comment:
Is this actually used? 🤔 I don't see any changes to the integration to use it? Am I right in thinking this is added _automatically_ by the otel integration, and you're just adding it here so that we get the right value for `PeerService`?

Reply:
> Am I right in thinking this is added automatically by the otel integration, and you're just adding it here so that we get the right value for PeerService

Yep, 100% this